teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -68,6 +152,43 @@ Every week, there are a couple essential things that every manager should take c
 
 This checklist should be pasted into each review, and checked off completely by Sunday.
 
+## _Weekly Code Review Documentation \[Michelle\]_
+
+Performing code reviews is an incredibly important step, and this function has to be done for every pull request for each of the developers every week. To start you dev should have requested a review from you. If not, please gently remind them or just request yourself. Then you should see a geen "leave a review" button and pressing that will lead you to a page displaying the code differences. You should leave comments on the code that you are viewing with specific things that they can improve on in their nect iteration.
+
+These specific changes should all contribute one way or another to the "Pull Request Checklist" as shown above. This checklist has to be pasted into each pull request. When reviewing, it is important to focus on both the "macro" and the "micro" issues that need to be resolved. This check list helps reviewers raise issues and review in a more efficient manner.
+
+Stemming from an effort to be more transparent and let developers be aware of how their work contributes to the organization as a whole, this list both ensures proper workflow and a more stylistically workflow. As more and more of the list is checked off, the developer is able to see their progress. Ideally, the list for a specific PR is completely checked off by the end of the week.
+
+## _Quality Assurance Procedures for Managers and Developers \[Jason\]_
+
+We want to the developers and the managers to provide give their input for each aspect of our projects. Therefore, we need procedures that allow the developers to quickly and effectively work together to complete tasks while maintaining a standard of quality.
+
+### Procedure:
+
+* [ ] Managers and Kevin conduct code reviews through GitHub
+  * [ ] Checking Issues and contributions
+    * Each time a developer makes a Pull Request, they need to provide a list of check boxes of items they addressed for their work that week.
+    * When managers review each Pull Request, they must check to ensure that all assigned issues in the milestone were completed and the checkbox contributions match the assigned issues.
+  * [ ] Every checkbox must be addressed with a comment by the reviewer
+    * A comment can acknowledge that the developer did something correctly or give constructive feedback.
+
+## _Revision by Head of Developer Relations and President \[Jeff\]_
```

Comment:
Can you be consistent as to whether you use "Bit" or "Bit Project"?

Also I can totally imagine a white lady in a pant suit walking slowly and gesturing with her sightly wrinkled middle aged hands in the foreground of who knows where because it may as well be a stock photo of some generic nature or car dealership.